crates/astria-sequencer/src/sequence/action.rs

```diff
@@ -1,10 +1,33 @@
+use anyhow::{
+    ensure,
+    Context,
+    Result,
+};
 use astria_proto::sequencer::v1alpha1::SequenceAction as ProtoSequenceAction;
 use serde::{
     Deserialize,
     Serialize,
 };
+use tracing::instrument;
+
+use crate::{
+    accounts::{
+        state_ext::{
+            StateReadExt,
+            StateWriteExt,
+        },
+        types::{
+            Address,
+            Balance,
+        },
+    },
+    transaction::action_handler::ActionHandler,
+};
+
+/// Fee charged for a sequence `Action` per byte of `data` included.
+pub(crate) const SEQUENCE_ACTION_FEE_PER_BYTE: Balance = Balance(1);
```

Comment:
See my comment re `calculate_sequencer_fee`: I think I'd prefer to make this non-pub(crate) and instead have a utility that calculates the fee.

Also, calling `Balance` a `Fee` is increasingly confusing.

Reply:
> Also, calling Balance a Fee is increasingly confusing.

what would be preferred? having just a uint? I think it's ok because the fee is always in terms of an account balance (ie deducting from it)

Reply:
Maybe we should literally start using something like `token`?